other/22e4e4f/22e4e4f.bug

```diff
@@ -47,5 +47,5 @@ fault-codes:
   - PROGRAMMING:BROKEN-CONTRACT
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```

Comment:
What's the reason for MONITORING on this one?

Reply:
same as https://github.com/robust-rosin/robust/pull/441#discussion_r682619842

Reply:
I have no idea why I put this comment here. This is an obvious MONITORING issue :facepalm: